### PR TITLE
Add Font Size choice

### PR DESCRIPTION
**Pull request by LordJZ**
This is an implementation for #830.

I wonder if it is possible to specify the "github default" as default. In other words, make the default such that github's css values are used. I couldn't find a way to do this, so I used `12px` as the default.

### CODE REVIEW

**Comment by the-j0k3r** [comment]
>  In other words, make the default such that github's css values are used. I couldn't find a way to do this, so I used `12px` as the default.

Hi thx for PR, I think its a good addition =), but I also wonder the same thing, lets wait for a master of the universe to add something on that.

**Comment by silverwind** [comment]
> make the default such that github's css values are used

Hard, but not impossible. It could be done in `tools/generate.js`, but I don't think it would be worth it given how unlikely it is that GitHub will change their code font size.

Changes look good and throurgh to me, but maybe @Mottie can have a final look.

File: defaults.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "themeJP"  : "twilight",
   "color"    : "#4f8cc9",
   "font"     : "Menlo",
+  "fontSize" : "12px",
```
Comment (LordJZ) [inline defaults.json]:
```suggestion
  "fontSize" : "github default",
```

File: github-dark.user.css [hunk 2]
```diff
@@ -172,6 +172,7 @@
   } EOT;
 }
 @advanced text font-choice "Code Font" "Menlo"
+@advanced text font-size-choice "Code Font Size" "12px"
```
Comment (LordJZ) [inline github-dark.user.css]:
```suggestion
@advanced text font-size-choice "Code Font Size" "github default"
```

**Comment by LordJZ** [comment]
There you go @the-j0k3r, thanks.

**Comment by the-j0k3r** [comment]
> There you go @the-j0k3r, thanks.

np, Thank you =)

File: github-dark.user.css (hunk 2 above)
Comment (Mottie) [reply github-dark.user.css]: Did you try setting the font-size to `initial` or `inherit`?

**Comment by silverwind** [comment]
> font-size to initial or inherit

I don't think this will work because it would be a valid rule and would potentially override other rules on the same object. I think a invalid rule is the best way to go for the default.

**Comment by LordJZ** [comment]
> 
> 
> > font-size to initial or inherit
> 
> I don't think this will work because it would be a valid rule and would potentially override other rules on the same element. I think a invalid rule is the best way to go for the default.

Yep -- that's why the default is `default` which is an invalid value for `font-size`.

**Comment by Mottie** [comment]
Derp!

**Pull request merged**